src/rp2_common/hardware_sync/include/hardware/sync.h

```diff
@@ -54,26 +70,39 @@ typedef volatile uint32_t spin_lock_t;
 #define PICO_SPINLOCK_ID_HARDWARE_CLAIM 11
 #endif
 
-// PICO_CONFIG: PICO_SPINLOCK_ID_STRIPED_FIRST, Spinlock ID for striped first, min=16, max=31, default=16, group=hardware_sync
+// PICO_CONFIG: PICO_SPINLOCK_ID_OS1, Spinlock ID reserved for use by low level OS style software, min=0, max=31, default=14, group=hardware_sync
+#ifndef PICO_SPINLOCK_ID_OS1
+#define PICO_SPINLOCK_ID_OS1 14
+#endif
+
+// PICO_CONFIG: PICO_SPINLOCK_ID_OS2, Spinlock ID reserved for use by low level OS style software, min=0, max=31, default=15, group=hardware_sync
```

Comment:
`PICO_SPINLOCK_ID_OS1` has an identical description to `PICO_SPINLOCK_ID_OS2` which `tools/extract_configs.py` isn't happy about. (I added this check because when we were first adding these `PICO_CONFIG` entires, there were a lot of copy'n'paste errors).
So you either need to make the descriptions slightly different, or you (or I?) need to take that "no duplicate descriptions" check out of `tools/extract_configs.py`.

Reply:
fixed